awx/ui_next/src/components/Schedule/shared/ScheduleForm.jsx

```diff
@@ -110,22 +119,37 @@ function ScheduleFormFields({ hasDaysToKeepField, zoneOptions }) {
       />
       <FormGroup
         fieldId="schedule-start-datetime"
-        helperTextInvalid={startDateTimeMeta.error}
+        helperTextInvalid={
+          startDateMeta.error ||
+          startDateMeta.error ||
+          startTimeMeta.error ||
+          startTimeMeta.error
+        }
         isRequired
         validated={
-          !startDateTimeMeta.touched || !startDateTimeMeta.error
+          !startDateMeta.touched ||
+          !startDateMeta.error ||
+          startTimeMeta.error ||
+          startTimeMeta.error
             ? 'default'
             : 'error'
         }
         label={t`Start date/time`}
       >
-        <input
-          className="pf-c-form-control"
-          type="datetime-local"
-          id="schedule-start-datetime"
-          step="1"
-          {...startDateTime}
-        />
+        <DateTimeGroup>
```

Comment:
I could _maybe_ see it being useful to move this out to it's own component if it makes it simpler to test.

@AlexSCorey Does the API expect a `datetime` for these or separate `date` and `time`?  

* If the api expects a `datetime`, we'd probably want some extra test coverage on whatever code we're using to stitch the input values together before presenting them to the api.
* If the api expects separate `date` and `time` values, I like this PR even more :)